config/locales/en.yml

```diff
@@ -305,6 +305,16 @@ en:
     user_char_added: Userspace chars added
     user_ns: User
 
+  recent_activity:
+    show_courses: Show My Courses Only
+    no_eligible: There are not currently any DYK-eligible articles.
```

Comment:
This key should be more specific. Without reading the message, it's not clear that it's specifically about DYK.